buildstockbatch/postprocessing.py

```diff
@@ -607,6 +608,9 @@ def upload_results(aws_conf, output_dir, results_dir):
         all_files.append(file.relative_to(parquet_dir))
     for file in [*ts_dir.glob('_common_metadata'), *ts_dir.glob('_metadata')]:
         all_files.append(file.relative_to(parquet_dir))
+    buildstock_csv = []   
+    for file in buildstock_dir.glob('buildstock.csv'):
+        buildstock_csv.append(file.relative_to(buildstock_dir))
```

Comment:
I see why you made this a list so you could use `map` below, but that's not necessary. You can just pull the filename here. We should also only upload it if it exists. I think ComStock might put this file somewhere different, so it may not always be where you're looking for it.

```suggestion
    buildstock_csv = buildstock_dir / 'buildstock.csv'
```

I haven't tested that this works. I might be missing some normalization or something.